bundles/org.openhab.binding.kostalinverter/src/main/resources/OH-INF/thing/Channels.xml

```diff
@@ -484,4 +484,431 @@
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>Current Dir</label>
+		<description>Current dir</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.0f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-charge-cycles">
+		<item-type>String</item-type>
+		<label>Charge Cycles</label>
+		<description>Total number of charge cycles</description>
+		<category>Energy</category>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="device-local-battery-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Battery Temperature</label>
+		<description>Current battery temperature</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-loginterval">
+		<item-type>Number:Time</item-type>
+		<label>Log Interval</label>
+		<description>Set value for loginterval</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.0f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-s0-inpulse-cnt">
+		<item-type>String</item-type>
+		<label>S0 Input Pulses Counter</label>
+		<description>S0-pulse counter</description>
+		<category>Energy</category>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="statistic-own-cons-rate-total">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Own Cons Rate Total</label>
+		<description>Total own comsumption rate</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.0f %unit%"/>
+	</channel-type>
+	<channel-type id="statistic-autonomy-degree-total">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Autonomy Degree Total</label>
+		<description>Total autonomy degree</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.0f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-battery-voltage-second-gen">
+		<item-type>Number:ElectricalPotential</item-type>
+		<label>Battery Voltage</label>
+		<description>Current battery voltage</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.0f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-bat-state-of-charge">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Bat State Of Charge</label>
+		<description>Current battery charge state</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.0f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-self-consumption">
+		<item-type>Number:Power</item-type>
+		<label>Self Consumption</label>
+		<description>Current self consumption</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-charge-time-end">
+		<item-type>Number:Time</item-type>
+		<label>Charge Time End</label>
+		<description>Set charge time end</description>
+		<category>Energy</category>
+		<state readOnly="true"/>
```

Comment:
According the readme, this and the following Channels are writeable. Should this be false?